### PR TITLE
DO-194: Family details accordion UX

### PR DESCRIPTION
**Pull request by amitjain24524**

### CODE REVIEW

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx [hunk 1]
```diff
@@ -41,9 +40,9 @@ const ArrayTemplateForm = (props) => {
   const renderHeading = () => {
     // FIXME: Should use Paragraph component of Typography
     if (element.children.props.formData.name) {
-      return <p>{heading}<Icon icon={isEditClicked ? faMinus : faPlus} /></p>;
+      return <div className="form-expander-row"><div className="form-expander-row__heading">{heading}</div><span className="form-expander-row__icon"><Icon icon={isEditClicked ? faMinus : faPlus} /><ArrayTemplateRemoveButton element={element} /></span></div>;
```
Comment (mohitsaladiya) [inline src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx:43]: Please follow class naming convention strictly.
```suggestion
      return <div className="array-template-form__form-expander-row"><div className="array-template-form__form-expander-row-heading">{heading}</div><span className="array-template-form__form-expander-row-icon"><Icon icon={isEditClicked ? faMinus : faPlus} /><ArrayTemplateRemoveButton element={element} /></span></div>;
```

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx [hunk 2]
```diff
@@ -41,9 +40,9 @@ const ArrayTemplateForm = (props) => {
...
     }
-    return <p><Icon icon={isEditClicked ? faMinus : faPlus} /></p>;
+    return <div className="form-expander-row"><div className="form-expander-row__heading no-data">New family member</div><span className="form-expander-row__icon"><Icon icon={isEditClicked ? faMinus : faPlus} /><ArrayTemplateRemoveButton element={element} /></span></div>;
```
Comment (mohitsaladiya) [inline src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx:45]:
```suggestion
    return <div className="array-template-form__form-expander-row"><div className="array-template-form__form-expander-row__heading no-data">New family member</div><span className="array-template-form__form-expander-row__icon"><Icon icon={isEditClicked ? faMinus : faPlus} /><ArrayTemplateRemoveButton element={element} /></span></div>;
```

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx [hunk 3]
```diff
@@ -52,12 +51,12 @@ const ArrayTemplateForm = (props) => {
   };
 
   return (
-    <>
+    <div className="form-expander">
```
Comment (mohitsaladiya) [inline src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx:54]:
```suggestion
    <div className="array-template-form__form-expander">
```

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/CollapsibleArrayTemplate.jsx [hunk 4]
```diff
@@ -45,7 +45,7 @@ const CollapsibleArrayTemplate = (props) => {
 
   const renderTitle = () => {
     if (title && title.length) {
-      return <p>{title}</p>;
+      return <p className="form-expander-heading">{title}</p>;
```
Comment (mohitsaladiya) [inline src/components/common/Typography/Form/CollapsibleArrayTemplate/CollapsibleArrayTemplate.jsx:48]:
```suggestion
      return <p className="collapsible-array-template__form-expander-heading">{title}</p>;
```

File: src/components/member/MemberDetailsWizard/CustomForm/FamilyDetailsForm/FamilyDetailsForm.jsx [hunk 5]
```diff
@@ -83,7 +83,7 @@ class FamilyDetailsForm extends React.Component {
   };
 
   renderAddButton = () => (
-    <Button value="Add" onClick={this.onClickAdd} disabled={this.state.addButtonDisabled} />
+    <Button value="Add" className="button__smallest button__blue" onClick={this.onClickAdd} disabled={this.state.addButtonDisabled} />
```
Comment (mohitsaladiya) [inline src/components/member/MemberDetailsWizard/CustomForm/FamilyDetailsForm/FamilyDetailsForm.jsx:86]:
```suggestion
    <Button value="Add" className="button__smallest button__blue" onClick={this.onClickAdd} disabled={this.state.addButtonDisabled} />
```
```suggestion
    <Button value="Add" className="family-details-form__smallest button__blue" onClick={this.onClickAdd} disabled={this.state.addButtonDisabled} />
```

File: src/components/member/MemberDetailsWizard/CustomForm/FamilyDetailsForm/FamilyDetailsForm.jsx [hunk 6]
```diff
@@ -107,7 +107,7 @@ class FamilyDetailsForm extends React.Component {
     return (
       <div className="detail-form">
         {isEditable ? this.renderEditButton() : null}
-        {this.renderAddButton()}
+        <div className="add-button-container"><div className="add-button-content">{this.renderAddButton()}</div></div>
```
Comment (mohitsaladiya) [inline src/components/member/MemberDetailsWizard/CustomForm/FamilyDetailsForm/FamilyDetailsForm.jsx:110]:
```suggestion
        <div className="family-details-form__add-button-container"><div className="family-details-form__add-button-content">{this.renderAddButton()}</div></div>
```

File: src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileEditView/MemberProfileEditView.jsx [hunk 7]
```diff
@@ -128,7 +128,7 @@ class MemberProfileEditView extends React.Component {
       // FIXME: TODO we need to break down it into components
       //  with dynamic values currently it's not configurable
       return (
-        <div key={memberDetails.id}>
+        <div key={memberDetails.id} className="member-profile-edit">
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileEditView/MemberProfileEditView.jsx:131]:
```suggestion
        <div key={memberDetails.id} className="member-profile-edit-view">
```

File: src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileEditView/_MemberProfileEditView.scss [hunk 8]
```diff
@@ -22,535 +22,605 @@
...
-              @media(max-width: 1024px) {
-                font-size: 28px;
-              }
-
-            }
-          }
-        }
-    .mbr-dtl-wzrd__bttn-bar {
+  }
+  .form-expander-heading {
```
Comment (mohitsaladiya) [inline src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileEditView/_MemberProfileEditView.scss:576]: This does not belong here as it should be a part of _CollapsibleArrayTemplate.scss

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx (hunk 2 above)
Comment (amitjain24524) [reply src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx]: Done

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx (hunk 1 above)
Comment (amitjain24524) [reply src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx]: Done

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx (hunk 3 above)
Comment (amitjain24524) [reply src/components/common/Typography/Form/CollapsibleArrayTemplate/ArrayTemplateForm/ArrayTemplateForm.jsx]: Done

File: src/components/common/Typography/Form/CollapsibleArrayTemplate/CollapsibleArrayTemplate.jsx (hunk 4 above)
Comment (amitjain24524) [reply src/components/common/Typography/Form/CollapsibleArrayTemplate/CollapsibleArrayTemplate.jsx]: Done

File: src/components/member/MemberDetailsWizard/CustomForm/FamilyDetailsForm/FamilyDetailsForm.jsx (hunk 5 above)
Comment (amitjain24524) [reply src/components/member/MemberDetailsWizard/CustomForm/FamilyDetailsForm/FamilyDetailsForm.jsx:86]: button__smallest is a common class for all button.

File: src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileEditView/_MemberProfileEditView.scss (hunk 8 above)
Comment (amitjain24524) [reply src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileEditView/_MemberProfileEditView.scss]: Done

File: src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileEditView/MemberProfileEditView.jsx (hunk 7 above)
Comment (amitjain24524) [reply src/components/routes/Member/AuthenticatedMember/MemberProfileViewRoute/MemberProfileEditView/MemberProfileEditView.jsx]: Done

File: src/components/member/MemberDetailsWizard/CustomForm/FamilyDetailsForm/FamilyDetailsForm.jsx (hunk 6 above)
Comment (amitjain24524) [reply src/components/member/MemberDetailsWizard/CustomForm/FamilyDetailsForm/FamilyDetailsForm.jsx]: Done

**Pull request merged**